src/Components/ContextProvider/ContextProvider.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  FC,
+  ReactNode,
+  useState,
+  useEffect,
+  useMemo,
+} from 'react';
+
+import {
+  getTodos,
+  updateTodo,
+} from '../../api/todos';
+
+import { Context } from '../../Context';
+
+import { Todo } from '../../types/Todo';
+import { ErrorMessage } from '../../types/ErrorMessage';
+import { Filter } from '../../types/Filter';
+
+interface Props {
+  children: ReactNode;
+}
+
+export const ContextProvider: FC<Props> = ({ children }) => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Filter>(Filter.ALL);
+  const [
+    errorMessage,
+    setErrorMessage,
+  ] = useState('');
+
+  const USER_ID = 12176;
+
+  const handleErrorChange = (value: string) => {
+    setErrorMessage(value);
+  };
```

Comment:
redundant function - just use `setErrorMessage` everywhere instead of `handleErrorChange`